### PR TITLE
Kernel update and change the watchdog ping to 35 seconds (on the 1.X branch)

### PR DESCRIPTION
**Pull request by floion**
@agherzan @michal-mazurek

### CODE REVIEW

**Comment by agherzan** [comment]
needs one for 2.X too

**Comment by floion** [comment]
It will follow, this is not even complete yet.

**Comment by agherzan** [comment]
Uh ok!

**Comment by floion** [comment]
When it is complete I will add people for reviewing

File: layers/meta-resin-edison/recipes-core/systemd/systemd_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
```
Comment (agherzan) [inline layers/meta-resin-edison/recipes-core/systemd/systemd_%.bbappend]: Same comment from the other PR: FILESEXTRAPATHS_prepend would suffice.

**Comment by floion** [comment]
fixes https://github.com/resin-os/resinos/issues/139

**Pull request merged**